[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_store with id: 2024-12-04_11-44-46__3dab62f2-6563-438c-9dc5-61b7ff290cab
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.light_push:light_push.py:28 Running fixture setup: light_push_setup
[35mDEBUG   [0m src.steps.relay:relay.py:28 Running fixture setup: relay_setup
[35mDEBUG   [0m src.steps.store:store.py:31 Running fixture setup: store_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-12-04_11-44-46__3dab62f2-6563-438c-9dc5-61b7ff290cab__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.124.65
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['9200', '9201', '9202', '9203', '9204']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 9200:9200 -p 9201:9201 -p 9202:9202 -p 9203:9203 -p 9204:9204 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=9202 --rest-port=9200 --tcp-port=9201 --discv5-udp-port=9203 --rest-address=0.0.0.0 --nat=extip:172.18.124.65 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=cceb4c995de593d5c028614beeaabe7e1cd34a67b82d5f2a0dece9ab08185cb5 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=9204 --metrics-logging=true --relay=true --store=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.124.65 waku f561928a91f0f2b3d2d4a109c8a3bf6573125c01e41cfa4100beb03aa8d7b382
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID f561928a91f0. Setting up logs at ./log/docker/node1_2024-12-04_11-44-46__3dab62f2-6563-438c-9dc5-61b7ff290cab__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 9200
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:9200/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:9200/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.124.65/tcp/9201/p2p/16Uiu2HAm6NbhMhjUKQxPaQ6rmXeCqXVupK29Tr3Cw4qQ2YzNVoUH","/ip4/172.18.124.65/tcp/9202/ws/p2p/16Uiu2HAm6NbhMhjUKQxPaQ6rmXeCqXVupK29Tr3Cw4qQ2YzNVoUH"],"enrUri":"enr:-L24QIAHPazgTdaSCOrxOpyhx3u6Mneq_raKaErNuRLU5c9YGjbhlWpfs3f1_ENKLPsWoBsSBORotstKI0P2ScI8HSACgmlkgnY0gmlwhKwSfEGKbXVsdGlhZGRyc5YACASsEnxBBiPxAAoErBJ8QQYj8t0DgnJzhQADAQAAiXNlY3AyNTZrMaECoqb4dPgl4d2QSSY3Hf3CSxyl53VB5x3Zmvfk0LW1ETaDdGNwgiPxg3VkcIIj84V3YWt1MgM"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-12-04_11-44-46__3dab62f2-6563-438c-9dc5-61b7ff290cab__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.117.228
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['24860', '24861', '24862', '24863', '24864']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 24860:24860 -p 24861:24861 -p 24862:24862 -p 24863:24863 -p 24864:24864 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=24862 --rest-port=24860 --tcp-port=24861 --discv5-udp-port=24863 --rest-address=0.0.0.0 --nat=extip:172.18.117.228 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=ab08ec508bee783cae62e717bcda2e25adde7bffaeed0d2eafaf1f3bd8b0bef9 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=24864 --metrics-logging=true --relay=true --store=true --discv5-bootstrap-node=enr:-L24QIAHPazgTdaSCOrxOpyhx3u6Mneq_raKaErNuRLU5c9YGjbhlWpfs3f1_ENKLPsWoBsSBORotstKI0P2ScI8HSACgmlkgnY0gmlwhKwSfEGKbXVsdGlhZGRyc5YACASsEnxBBiPxAAoErBJ8QQYj8t0DgnJzhQADAQAAiXNlY3AyNTZrMaECoqb4dPgl4d2QSSY3Hf3CSxyl53VB5x3Zmvfk0LW1ETaDdGNwgiPxg3VkcIIj84V3YWt1MgM --storenode=/ip4/172.18.124.65/tcp/9201/p2p/16Uiu2HAm6NbhMhjUKQxPaQ6rmXeCqXVupK29Tr3Cw4qQ2YzNVoUH
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.117.228 waku cd0ddc6f96770ae317a820c182585c7ea8454419e49a1ea9087b5d6dc8f5a07e
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID cd0ddc6f9677. Setting up logs at ./log/docker/node1_2024-12-04_11-44-46__3dab62f2-6563-438c-9dc5-61b7ff290cab__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 24860
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:24860/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:24860/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.117.228/tcp/24861/p2p/16Uiu2HAmDwQhDZfyZwNucuQroiqBsYwUSgT6nhVbsParyVgNa1sn","/ip4/172.18.117.228/tcp/24862/ws/p2p/16Uiu2HAmDwQhDZfyZwNucuQroiqBsYwUSgT6nhVbsParyVgNa1sn"],"enrUri":"enr:-L24QH3KQiu_G_iCa7EZMuNoYtzq9m1nA2cImhz-3BWnE9fbUeEkoUk7Ac0cBiASZXD6Q3-YoHKEiJzM7ZUrVR55kRYCgmlkgnY0gmlwhKwSdeSKbXVsdGlhZGRyc5YACASsEnXkBmEdAAoErBJ15AZhHt0DgnJzhQADAQAAiXNlY3AyNTZrMaEDExCBV6JUuQ7V6PDo2S0nBb7q41lZ0H-IlnJf-vWmtlmDdGNwgmEdg3VkcIJhH4V3YWt1MgM"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9200/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:24860/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9200/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "UmVsYXkgd29ya3MhIQ==", "contentTopic": "/test/1/waku-relay/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:218 Checking that peer wakuorg/nwaku:latest can find the stored messages
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:24860/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F1&pageSize=5&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"","statusCode":200,"statusDesc":"OK","messages":[]}'
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9200/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "UmVsYXkgd29ya3MhIQ==", "contentTopic": "/test/1/waku-relay/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:218 Checking that peer wakuorg/nwaku:latest can find the stored messages
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:24860/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F1&pageSize=5&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"","statusCode":200,"statusDesc":"OK","messages":[]}'
[35mDEBUG   [0m src.steps.store:store.py:132 Relaying message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:9200/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "UmVsYXkgd29ya3MhIQ==", "contentTopic": "/test/1/waku-relay/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.2 seconds
[35mDEBUG   [0m src.steps.store:store.py:218 Checking that peer wakuorg/nwaku:latest can find the stored messages
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:24860/store/v3/messages?pubsubTopic=%2Fwaku%2F2%2Frs%2F3%2F1&pageSize=5&ascending=true" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"","statusCode":200,"statusDesc":"OK","messages":[{"messageHash":"TwgxSaaDITrMxwH2OiVyI2StVCv9K9fbXknFUgTFG6g="}]}'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id f561928a91f0
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id cd0ddc6f9677
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container f561928a91f0. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.